lib/rules/jsx-no-bind.js

```diff
@@ -5,13 +5,25 @@
  */
 'use strict';
 
+var Components = require('../util/Components');
 var propName = require('jsx-ast-utils/propName');
 
 // -----------------------------------------------------------------------------
 // Rule Definition
 // -----------------------------------------------------------------------------
 
-module.exports = function(context) {
+function renderReturnsJSX(node, utils) {
```

Comment:
I think it makes sense for "is returning JSX" to accept any kind of function or `class`, and it should be able to accurately return a boolean.